packages/inter-protocol/test/vaultFactory/driver.js

```diff
@@ -73,7 +74,7 @@ const defaultParamValues = debt =>
  * @typedef {{
  * aeth: IssuerKit & import('../supports.js').AmountUtils,
  * aethInitialLiquidity: Amount<'nat'>,
- * puppetGovernors: { [contractName: string]: ERef<import('@agoric/governance/tools/puppetContractGovernor').PuppetContractGovernorKit<any>['creatorFacet']> },
+ * puppetGovernors: { [contractName: string]: ERef<import('@agoric/governance/tools/puppetContractGovernor.js').PuppetContractGovernorKit<any>['creatorFacet']> },
```

Comment:
not necessary in TS imports. on the plus side the extension helps with find-replace.